### PR TITLE
Layout changes

### PR DESCRIPTION
**Pull request by merowin**
### Proposed Changes
- Pull pageWrapper into each View
- in Home View, move the games list into the right column
- in Game View, add Divs for left and right column

### Reasoning
With the first change, it is easier for each view to define its layout individually. This also helps with the second change.
In the right column, the games list is easier accessible on desktop. For mobile it moves below as usual.
In the Game view, previously each added html element was positioned as a new element in the grid layout, resulting in elements being positioned seemingly randomly. I think it makes more sense to have a left- and right column that encapsulate all other elements.

### Screenshots
grafik

grafik

grafik

grafik

### CODE REVIEW

**Comment by JonKo314** [comment]
> Pull pageWrapper into each View

> With the first change, it is easier for each view to define its layout individually.

This is bugging me a bit. Why does this help? Could the same be achieved by applying the necessary styling to the `<main>` element?

```html
<div class="pageWrapper">
  <Suspense><RouterView /></Suspense>
</div>
```

"Everything" should already be wrapped inside a `.pageWrapper` `<div>`, because the `<RouterView>` is wrapped inside it.

**Comment by merowin** [comment]
> "Everything" should already be wrapped inside a `.pageWrapper` `<div>`, because the `<RouterView>` is wrapped inside it.

Yes, but what if a View doesn't want to have a grid layout, or doesn't want to have the pageWrapper as the outmost element?

For example, with the status quo I can't pull the games list into the right column, unless I pull it out of the `<main>` element.

> Could the same be achieved by applying the necessary styling to the `<main>` element?

I'm not sure what you mean.

**Comment by JonKo314** [comment]
I guess our styling structure is not very thought through. 🤔 

I'm okay with doing what works, maybe I find the motivation to come up with a better concept at some point.

**Comment by merowin** [comment]
> I guess our styling structure is not very thought through. 🤔

To be honest I'm not good at layouting, or styling in general. But I can see that there is much that can be improved about the layout / styling on our site. So much that I think maybe even I can improve it 😝

File: packages/vue-client/src/views/ComponentView.vue [hunk 1]
```diff
@@ -52,40 +52,44 @@ function update_board_state(pos: Coordinate) {
...
-    <div>
-      <label>Radius:</label>
-      <input type="number" v-model="r" />
-    </div>
-    <div>
-      <label>cx:</label>
-      <input type="number" v-model="cx" />
-      <label>cy:</label>
-      <input type="number" v-model="cy" />
+    <div class="PageWrapper">
```
Comment (benjaminpjones) [inline packages/vue-client/src/views/ComponentView.vue:55]: Not sure if css cares, but better to be consistent

```suggestion
    <div class="pageWrapper">
```

**Review by benjaminpjones — APPROVED** [review]
It's definitely a nice visual improvement!

I wonder if it would help to be a bit more descriptive about what `pageWrapper` entails, now that we're not using it for *every* page.  Something like "grid-page-layout" or similar.  (I also slipped in kebab-case, I think this is more widespread convention for CSS)

File: packages/vue-client/src/views/AboutView.vue [hunk 2]
```diff
@@ -1,18 +1,22 @@
 <template>
   <main>
-    <h2>Who are we?</h2>
-    <p>
-      <span>Check us out on </span>
-      <a href="https://github.com/benjaminpjones/govariants">Github</a>
```
[the remark this answers is not in the document]
Comment (merowin) [reply packages/vue-client/src/views/AboutView.vue:6]: updated to `https://github.com/govariantsteam/govariants`

**Comment by merowin** [comment]
> I wonder if it would help to be a bit more descriptive about what `pageWrapper` entails, now that we're not using it for _every_ page. Something like "grid-page-layout" or similar. (I also slipped in kebab-case, I think this is more widespread convention for CSS)

I agree, renamed it to `grid-page-layout`

**Pull request merged**